ERROR:root:Received data is not in JSON format
Traceback (most recent call last):
  File "/usr/lib/cgi-bin/computeScheduleSolution.py", line 35, in parseJSONToDict
    jsonObject = json.loads(jsonText)
  File "/usr/lib/python3.5/json/__init__.py", line 319, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.5/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.5/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
